### PR TITLE
Add minor upgrade checklist

### PR DESCRIPTION
**Pull request by corvus-ch**

### CODE REVIEW

File: docs/modules/ROOT/pages/how-tos/new_minor.adoc [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+* API versions of resource types used
+* Alert rules
++
+The main suspect is `openshift4-monitoring`.
+But also lookout for components that define alert rules but are no longer picked up by cluster monitoring.
+
+. Conduct install on all supported cloud providers
++
+Use the installation documentation to install a new cluster.
+Do so for each supported hosting provider.
```
Comment (corvus-ch) [inline docs/modules/ROOT/pages/how-tos/new_minor.adoc:45]: Where is this defined?

**Review by simu — APPROVED** [review]
LGTM. Instead of leaving a pile of English suggestions, I directly pushed a commit which primarily adjusts wording.

File: docs/modules/ROOT/pages/how-tos/new_minor.adoc [hunk 2]
```diff
@@ -0,0 +1,50 @@
+= Checklist to prepare for a new minor version of OpenShift 4
+
+. Read and summarize release notes
++
+Read the release notes.
+Look for things that affect how we use OpenShift.
+Ensure each change which impacts us gets addressed.
++
+Write a summary and add it to xref:oc4:ROOT:references/release_notes.adoc[our release notes summary].
```
Comment (tobru) [inline docs/modules/ROOT/pages/how-tos/new_minor.adoc:9]: And remove OpenShift versions which we don't support anymore (usually current version - 1, but I suggest keeping current version - 2)

File: docs/modules/ROOT/pages/how-tos/new_minor.adoc [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+====
++
+[TIP]
+====
+Instead of using an existing cluster, you can also set up one with version `4.<n-1>` and then upgrade this one.
+====
+
+. Check compatibility of components
++
+Check if all core Commodore components are compatible with the new version.
```
Comment (tobru) [inline docs/modules/ROOT/pages/how-tos/new_minor.adoc:31]: What are the "core Commodore components"? Where is this defined?

File: docs/modules/ROOT/pages/how-tos/new_minor.adoc [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+Address all issues as necessary.
+If it's straight-forward, directly create a fix.
+Otherwise, create a follow-up story to address the issue.
++
+The best way to find an address issues is to inspect an upgraded and or newly installed cluster using the new minor version.
```
Comment (tobru) [inline docs/modules/ROOT/pages/how-tos/new_minor.adoc:36]:
```suggestion
The best way to find and address issues is to inspect an upgraded and or newly installed cluster using the new minor version.
```

File: docs/modules/ROOT/pages/how-tos/new_minor.adoc [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+Things to look for:
++
+* API versions of resource types used
+* Alert rules
+* Logic which generates configuration based on the cluster's OpenShift version.
++
+One component which definitely needs to be updated for a new minor version is `openshift4-monitoring`.
```
Comment (tobru) [inline docs/modules/ROOT/pages/how-tos/new_minor.adoc:43]:
```suggestion
One component which definitely needs to be updated for a new minor version is https://github.com/appuio/component-openshift4-monitoring[`openshift4-monitoring`].
```

File: docs/modules/ROOT/pages/how-tos/new_minor.adoc (hunk 1 above)
Comment (tobru) [reply docs/modules/ROOT/pages/how-tos/new_minor.adoc:49]: IMHO here: https://products.docs.vshn.ch/products/appuio/managed/ocp4.html (yes, this list needs to be updated)

**Review by tobru — APPROVED** [review]
A few minor things, and then we're good to go. Thanks.

**Review by simu — APPROVED** [review]

File: docs/modules/ROOT/pages/how-tos/new_minor.adoc [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+====
++
+[TIP]
+====
+Instead of using an existing cluster, you can also set up one with version `4.<n-1>` and then upgrade this one.
+====
+
+. Check compatibility of components
++
+Check if all core Commodore components{fn-core-components} are compatible with the new version.
```
Comment (tobru) [inline docs/modules/ROOT/pages/how-tos/new_minor.adoc:33]:
```suggestion
Check if all core Commodore components footnote:[Core Commodore components are those included by the distribution `openshift4` and its parent hierarchy levels.] are compatible with the new version.
```

There is no need for the complicated `{fn-core-components}` construct here...

**Review by tobru — APPROVED** [review]

File: docs/modules/ROOT/pages/how-tos/new_minor.adoc (hunk 6 above)
Comment (corvus-ch) [reply docs/modules/ROOT/pages/how-tos/new_minor.adoc:33]: I kind of disliked the interleaved sentences when looking at the asciidoc. It does not matter when rendered.

**Pull request merged**